### PR TITLE
    feat(metrics): exposed traffic metrics grouped by tenants, subscriptions, outputs

### PR DESCRIPTION
**Pull request by kristofgyuracz**
- **feat(metrics): exposed traffic metrics grouped by tenants, subscriptions, outputs**
- **update tests for changed config**

TODO:

- [x] Release proper collector image and use it

### CODE REVIEW

File: internal/controller/telemetry/otel_conf_gen.go [hunk 1]
```diff
@@ -312,12 +481,38 @@ func (cfgInput *OtelColConfigInput) generateProcessors() map[string]any {
 
 }
 
+func generateOutputExporterNameProcessor(output v1alpha1.OtelOutput) AttributesProcessor {
+	var exporterName string
+	if output.Spec.Loki != nil {
+		exporterName = fmt.Sprintf("loki/%s_%s", output.Namespace, output.Name)
+	} else if output.Spec.OTLP != nil {
+		exporterName = fmt.Sprintf("otlp/%s_%s", output.Namespace, output.Name)
+	}
```
Comment (pepov) [inline internal/controller/telemetry/otel_conf_gen.go:490]: we need to generate this somehow

File: internal/controller/telemetry/otel_conf_gen.go [hunk 2]
```diff
@@ -288,12 +415,52 @@ func (cfgInput *OtelColConfigInput) generateConnectors() map[string]any {
 
 }
 
+func generateCountConnectors() map[string]any {
```
Comment (pepov) [inline internal/controller/telemetry/otel_conf_gen.go:418]: we also need:
- namespace
- node name
- container name
- app label -> we have a "smart" solution for this in logging operator with syslog-ng, but I'm not sure we can replicate that with otel

Comment (kristofgyuracz) [reply internal/controller/telemetry/otel_conf_gen.go:439]: Working on a change to enable count connector to count by resource attributes, to be able to do this without dirty workarounds.

**Review by OverOrion — COMMENTED** [review]
LGTM! :rocket:

```
# HELP otelcollector_output_log_count_total The number of logs sent out from each exporter.
# TYPE otelcollector_output_log_count_total counter
otelcollector_output_log_count_total{exporter="otlp/collector_otlp-openobserve",subscription="subscription-sample-1",tenant="example-tenant"} 97
otelcollector_output_log_count_total{exporter="otlp/collector_otlp-openobserve",subscription="subscription-sample-2",tenant="example-tenant"} 97
# HELP otelcollector_tenant_log_count_total The number of logs from each tenant pipeline.
# TYPE otelcollector_tenant_log_count_total counter
otelcollector_tenant_log_count_total{tenant="example-tenant"} 97
```

**Review by OverOrion — APPROVED** [review]

**Pull request merged**